### PR TITLE
lint: remove duplicate staticcheck - covered already by golangci-lint

### PR DESCRIPTION
**Pull request by darwinz**

### CODE REVIEW

File: go/lint-project.sh [hunk 1]
```diff
@@ -177,7 +159,7 @@ fi
 if [[ "$OS_NAME" != "windows" ]]; then
     wget -q -O - -q https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s v1.37.1
 
-    enabled="-E=bodyclose,exhaustive,rowserrcheck"
```
Comment (adamdecaf) [inline go/lint-project.sh:180]: Does `exhaustive` still run if we remove it from this `-E` flag?

Comment (darwinz) [reply go/lint-project.sh:180]: I thought we said since it's experimental to remove it

Comment (adamdecaf) [reply go/lint-project.sh]: Yes, but `exhaustive` was not being ran with the removal on Lint 180. I pushed a commit to enable it again.

Before:
```
INFO [lintersdb] Active 11 linters: [bodyclose deadcode gosimple govet ineffassign rowserrcheck staticcheck structcheck typecheck unused varcheck] 
```

After: 
```
INFO [lintersdb] Active 12 linters: [bodyclose deadcode exhaustive gosimple govet ineffassign rowserrcheck staticcheck structcheck typecheck unused varcheck] 
```

Comment (darwinz) [reply go/lint-project.sh]: Got it.  Thanks Adam!

**Pull request merged**